src/ansys/mapdl/core/_commands/prep7/keypoints.py

```diff
@@ -0,0 +1,1177 @@
+            are ignored (valid only in the GUI).
+
+        kp2 : str
+            Second keypoint.
+
+        kpnew : str
+            Number assigned to the new keypoint. Defaults to the lowest available keypoint number.
+
+        type_ : str
+            Type of input for ``VALUE``.
+
+            * ``RATIO`` - Value is the ratio of the distances between keypoints as follows: ( ``KP1`` -
+              ``KPNEW`` )/( ``KP1`` - ``KP2`` ).
+
+            * ``DIST`` - Value is the absolute distance between ``KP1`` and ``KPNEW`` (valid only if current
+              coordinate system is Cartesian).
+
+        value : str
+            Location of new keypoint, as defined by ``Type`` (defaults to 0.5). If ``VALUE`` is a ratio (
+            ``Type`` = RATIO) and is less than 0 or greater than 1, the keypoint is created on the extended
+            line. Similarly, if ``VALUE`` is a distance ( ``Type`` = DIST) and is less than 0 or greater
+            than the distance between ``KP1`` and ``KP2``, the keypoint is created on the extended line.
+
+        Returns
+        -------
+        int
+            Keypoint number of the generated keypoint.
+
+        Notes
+        -----
+
+        .. _KBETW_notes:
+
+        Placement of the new keypoint depends on the currently active coordinate system ( :ref:`csys` ). If
+        the coordinate system is Cartesian, the keypoint will lie on a straight line between ``KP1`` and
+        ``KP2``. If the system is not Cartesian (for example, cylindrical, spherical, etc.), the keypoint
+        will be located as if on a line (which may not be straight) created in the current coordinate system
+        between ``KP1`` and ``KP2``. Note that solid modeling in a toroidal coordinate system is not
+        recommended.
+
+        Examples
+        --------
+        Create a keypoint exactly centered between two keypoints.
+
+        >>> k0 = mapdl.k("", 0, 0, 0)
+        >>> k1 = mapdl.k("", 1, 0, 0)
+        >>> k2 = mapdl.kbetw(k0, k1)
+        >>> k2
+        3
+        """
+        command = f"KBETW,{kp1},{kp2},{kpnew},{type_},{value}"
+        return parse.parse_kpoint(self.run(command, **kwargs))
+
+    def kcenter(
+        self,
+        type_: str = "",
+        val1: str = "",
+        val2: str = "",
+        val3: str = "",
+        val4: str = "",
+        kpnew: str = "",
+        **kwargs,
+    ):
+        r"""Creates a keypoint at the center of a circular arc defined by three locations.
+
+        Mechanical APDL Command: `KCENTER <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_KCENTER.html>`_
+
+        Parameters
+        ----------
+        type_ : str
+            Type of entity used to define the circular arc. The meaning of ``VAL1`` through ``VAL4`` will vary depending on ``Type``. If ``Type`` = P, graphical picking is enabled and all remaining command fields are ignored (valid only in the GUI).
```

Comment:
Very long line?

Reply:
This is due to the fact a list is in the argument description. I took note of it in https://github.com/ansys/pyconverter-xml2py/issues/413.